### PR TITLE
Code Editor - Popup covering text select on hovering | Edit tab data resets when tab closed | Hotfix for InfoColorBar

### PR DESCRIPTION
**Pull request by milindk8**
# Description

1. Whenever we have to select something in the deployment template, the description gets popped up and it doesn’t let us change the cursor position
2. Edit tab resets data when the tab gets closed:-
 Global Configuration <  Clusters & environment, Git accounts, Container registries 
3. Hotfix for InfoColorBar

Fixes - https://github.com/devtron-labs/devtron/issues/2114

8FE36A5D-AC02-4E0F-8EB0-405BEC7381E3

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Checklist:

* [x] The title of the PR states what changed and the related issues number (used for the release note).
* [ ] Does this PR require documentation updates?
* [ ] I've updated documentation as required by this PR.
* [x] I have performed a self-review of my own code
* [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/components/dockerRegistry/Docker.tsx [hunk 1]
```diff
@@ -84,22 +84,27 @@ function CollapsedList({
...
                     <List.Logo>
-                        {' '}
                         <div className={'registry-icon ' + registryType}></div>
                     </List.Logo>
-                ) : (
+                ) : (collapsed &&
                     <List.Logo>
                         <Add className="icon-dim-24 fcb-5 vertical-align-middle" />
                     </List.Logo>
                 )}
```
Comment (sohel-rp) [inline src/components/dockerRegistry/Docker.tsx:103]: We can this condition now to something like
```
{id && <List.Logo>}
{!id && collapsed && <List.Logo>}
```

File: src/components/gitProvider/GitProvider.tsx [hunk 2]
```diff
@@ -237,11 +241,12 @@ function CollapsedList({
                             </span>
                         </div>
                     ) : (
-                        <Add className="icon-dim-24 fcb-5 vertical-align-middle" />
+                        collapsed && <Add className="icon-dim-24 fcb-5 vertical-align-middle" />
```
Comment (sohel-rp) [inline src/components/gitProvider/GitProvider.tsx:244]: same here

File: src/components/dockerRegistry/Docker.tsx (hunk 1 above)
Comment (milindk8) [reply src/components/dockerRegistry/Docker.tsx:104]: done

File: src/components/gitProvider/GitProvider.tsx (hunk 2 above)
Comment (milindk8) [reply src/components/gitProvider/GitProvider.tsx]: Done

**Review by shivani170 — APPROVED** [review]
Looks Good 👍

**Review by shivani170 — APPROVED** [review]

**Review by sohel-rp — APPROVED** [review]

**Pull request merged**